as/cNFT/assembly/market.ts

```diff
@@ -72,7 +75,9 @@ export function accept_bid(tokenId: string, bidder: string): void {
     env.promise_return(promisePrevOwner.id)
 
     // Transfer token to bidder
-    nft_transfer(tokenId, bidder)
+    const transferArgs: NftTransferArgs = { "token_id": tokenId, "bidder_id": bidder }
+    const promiseTransfer = ContractPromise.create(context.contractName, "nft_transfer", transferArgs, XCC_GAS, u128.Zero)
```

Comment:
I think this was the preferred solution anyway but just for confirmation: why it was not working calling it internally? Still cause of `@nearBindgen`?

Reply:
Yes all three functions weren't working because of `@nearBindgen`